beacon-chain/blockchain/receive_attestation.go

```diff
@@ -95,7 +95,9 @@ func (s *Service) spawnProcessAttestationsRoutine() {
 				return
 			case slotInterval := <-ticker.C():
```

Comment:
This ticker is also triggered at 0s, so either it's a bug or we need to update the PR title and description

Reply:
why?

Reply:
Nvm. I see the `if slotInterval.Interval > 0 {` below